core/src/tangram.cpp

```diff
@@ -243,6 +250,10 @@ void Map::queueSceneUpdate(const std::vector<SceneUpdate>& sceneUpdates) {
     impl->sceneUpdates.insert(impl->sceneUpdates.end(), sceneUpdates.begin(), sceneUpdates.end());
 }
 
+std::shared_ptr<Platform> Map::getPlatform() const {
```

Comment:
We have to fix ourselves some coding rules concerning this within the codebase. I stayed consistent with the rest of the public API here (`getPixelScale`, `getViewportHeight`, `getTilt`...).